http-client/src/test/groovy/io/micronaut/http/client/HttpHeadSpec.groovy

```diff
@@ -364,7 +364,7 @@ class HttpHeadSpec extends Specification {
         body == "success"
 
         cleanup:
-        client.close()
+        client.refresh()
```

Comment:
this one too

Reply:
Same as previous comment

Reply:
this one actually use one shared instance 
<img width="1403" alt="Screenshot 2024-11-07 at 23 46 33 copy" src="https://github.com/user-attachments/assets/e8084ee1-54da-4790-9e40-91afc9d9c750">